docs/installation/database/passing-configuration.rst

```diff
@@ -0,0 +1,427 @@
+
+ODC Configuration (Details)
+***************************
+
+Open Data Cube configuration files, supported configuration options, and basic use cases and defaults
+are described in :doc:`configuration`.
+
+The Open Data Cube uses configuration files and/or environment variables to determine how to connect to databases.
+
+Summary
+=======
+
+`Raw configuration`_ can be explicitly passed in, otherwise, configuration is read from a `configuration file`_.
+
+Data in a configuration file can be supplemented with configuration by `environment variable`_.
+
+One configuration can define multiple environments, so users must `choose one`_.
+
+The configuration engine in 1.9 is not 100% compatible with the previous configuration engine.  Advanced
+users and developers upgrading 1.8 systems should read the `migration notes`_.
+
+.. _`configuration file`: #file-configuration
+.. _`choose one`: #the-active-environment
+.. _`environment variable`: #generic-environment-variable-overrides
+.. _`migration notes`: #migrating-from-datacube-1-8
+
+1. File configuration
+---------------------
+
+.. highlight:: python
+
+If `raw configuration`_ is not passed in, ODC attempts to find a configuration file in the file system.
+
+`Only one`_ configuration file is read.
+
+If your previous practice was to extend a shared system configuration file with a local
+user configuration file, then you will now need to take a copy of the system configuration file,
+add your extensions to your copy, and ensure that the Open Data Cube reads from your
+modified file.
+
+.. _`Only one`: #Merging-multiple-config-files
+
+1a. Default Search Paths
+++++++++++++++++++++++++
+
+If config file paths have not passed in through any of the methods below, 1b. through
+1d., then the Open Data Cube checks the following paths in order, with the
+first readable file found being read:
+
+1. :file:`./datacube.conf`    (in the current working directory)
```

Comment:
configuration.rst line 35 is just an example - the most common location that will usually work.  I'm trying to keep this level of detail out of that document.  I've added another crosslink to this file, how's that?

Reply:
Ok, since `./datacube.conf` was the first path checked I assumed that was the preferred location for a configuration file to be, but that's fine. Crosslink is good, ty